test/e2e/tests/smartButton.js

```diff
@@ -41,10 +41,15 @@ module.exports = {
     const smartButton = browser.page.smartButton();
     smartButton
       .navigate()
+      .pause(2500)
```

Comment:
Is this just for FF? Can we do only with FF or at least add a comment to note why?

Reply:
It's for both Chrome and Firefox. It's to wait for loading external content of the Smart Box iFrame, which I've noticed can be slightly delayed. The test code traverses into that iFrame and clicks smart button to load the Offers frame. Thinking more if there was an event that it could listen for to know the Smart Box iFrame has been loaded that would be best?

Reply:
Here is a recording to show the delay on load of iFrame

https://user-images.githubusercontent.com/161528/141587180-068d65e8-011b-4c80-a472-61395c23e9c1.mp4

Reply:
I replaced `pause` with `waitForElementPresent` and `waitForElementVisible` in https://github.com/subscriptions-project/swg-js/pull/1997/commits/826278706de1962f846221472380af4c8f6c983e which should be more explicit what it is waiting for and should achieve same result. Similar to how we are doing in [contribution](https://github.com/subscriptions-project/swg-js/blob/main/test/e2e/tests/contribution.js#L24).

Reply:
@adolgachev 

In 9cc3122, changed the click callback to handle different return values for chrome/FF.
This was the most simple fix, and with these changes tests should now be passing in chrome and firefox.
Reverted and shelved previous code that re-wrote test.